io.github.saeugetier.photobooth.json

```
@@ -0,0 +1,85 @@
+{
+    "app-id": "io.github.saeugetier.photobooth",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.9",
+    "sdk": "org.kde.Sdk",
+    "command": "qtbooth",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--device=all",
+        "--filesystem=home",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--env=LD_LIBRARY_PATH=/app/lib64:/app/lib/x86_64-linux-gnu:$LD_LIBRARY_PATH"
```

```suggestion
        "--socket=fallback-x11"
```

It's is still incorrect and unecessary.